README.md

```diff
@@ -14,15 +14,8 @@ Teams spend significant time building parts of the design system (colors, variab
 
 [Learn more about FAST-DNA packages on wiki...](https://github.com/Microsoft/fast-dna/wiki/Packages)
 
-## Policies
-### Guiding principles, contributions, and governance
-Details including getting started can be found [on the wiki](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md).
-
-### Code of conduct
-Details are located [on the wiki](https://github.com/Microsoft/fast-dna/blob/master/CODE_OF_CONDUCT.md).
-
-### Accessibility
-FAST-DNA has partnered with Deque & Microsoft to build accessibility into our components ensuring we are WCAG 2.0 AA compliant. More about accessibility can be found [on the wiki](https://github.com/Microsoft/fast-dna/wiki/Accessibility).
+## Documentation
+Detailed documentation can be found on our [GitHub Wiki](https://github.com/Microsoft/fast-dna/wiki) which covers getting started, community policies, and guidance on contributing, developing, quality, and testing.
```

Comment:
In the past, we've tried to stay away from pronouns. Just want to verify that this is intentional?